usagov/vote-gov-nvrf-app
# Vote-135: NPM Audit

Ticket: [Vote-135](https://cm-jira.usa.gov/browse/VOTE-3135)

Description:  Updating NPM packages that were outdated 

- a follow up ticket as been made for updating react but needed package for USWDS react will need to be updated for this new version. 

Testing:
1. run `npm i` and `npm run build` and ensure that builds with no errors 
2. start up app and fill out form and check for any major regressions 
3. cd into `testing` and run `npm run cy:test`
